content/_changelogs/9.5.2.md

```diff
@@ -0,0 +1,42 @@
+## 9.5.2
+
+_Released 3/14/2022_
+
+**Bugfixes:**
+
+- Fixed an issue with `.type()` where click events could be fired on the
+  incorrect target element because the target focus changed within a key-down
+  event handler callback. Fixed in
+  [#20525](https://github.com/cypress-io/cypress/pulls/20525).
+- Fixed a regression in [9.5.0](/guides/references/changelog#9-5-0) where ANSI
+  colors were not removed from the FireFox warning message about the
+  `chromeWebSecurity` configuration option having no effect on the Firefox
+  browser. Fixes [#20496](https://github.com/cypress-io/cypress/issues/20496).
+- Updates were made to the pre-release build setup such that Cypress will use a
+  unique cache folder for each
+  [pre-release installation](https://docs.cypress.io/guides/getting-started/installing-cypress#Install-pre-release-version)
```

Comment:
This needs to be a relative link within the docs, so we don't end up in dead links later on.